src/classes/CountsManager.cls

```diff
@@ -0,0 +1,73 @@
+/**
+*  Assignment       :   Assignment-4th (ApexSession#4)
+*
+*  Purpose          :   Class to demonstrate two methods which will update number of Classes and number of student
+*						fields on school object.
+*
+*  Created Date     :   07/09/2022
+*
+*  Created By       :   Vinayak Banjara
+*
+*  Revision Logs    :   V_1.0 - Created
+*
+**/
+
+
+public without sharing class CountsManager
+{
+    public static void updateNoOfStudentsDetails(String schoolName)
+    {
+        List<School__c> schoolToBeUpdated = [SELECT Id, Name
```

Comment:
Use LIMIT 1 in the  SOQL Query, as you are expecting only 1 School record for the schoolName .